.github/workflows/ci.yml

```diff
@@ -28,10 +28,10 @@ jobs:
 
     steps:
       - name: Copy repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
```

Comment:
could a be a separate small PR since it doesn't relate to updating the user entity.